.github/DISCUSSION_TEMPLATE/help.yml

```diff
@@ -1,25 +1,36 @@
----
 body:
-  - type: textarea
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to start a new discussion!
+        
+        Before you post:
+        • Check if someone has already asked/answered your question in a previous discussion.
+        
+        When you're ready to post:
+        • Add tags to your thread (e.g. 'React', 'Vue', 'Vite') to make your issue clearer for other users.
+        
+  - type: textarea 
     attributes:
       label: Summary
-      description: How do you need help?
-      placeholder: I need help with...
-      validations:
+      description: How can we help?
+    validations:
         required: true
+        
   - type: textarea
     attributes:
       label: Additional information
-      description: Share your Storybook configuration (`main.js or ts`), any error messages, or relevant dependencies
+      description: |
+        Share Your Storybook configuration (`main.js or main.ts`), your Storybook version number, any error messages, or any relevant dependencies. These help us get a clearer understanding of what might be going wrong.
+
+        P.s. please share code as text rather than as a screenshot! It makes debugging much easier and faster.
       render: js
-      validations:
+    validations:
         required: false
+        
   - type: input
     attributes:
-      label: Share an example
-      description: Help us debug your issue by creating a minimal reproduction. You can do this with StackBlitz by heading to [https://storybook.new](https://storybook.new)!
+      label: Create a reproduction
+      description: Help us debug by creating a minimal reproduction with [https://storybook.new](https://storybook.new)
```

Comment:
Hey @vanessayuenn – I tried the edited code in a different forked version of Storybook and it looks like it works (pic below). 

I can't get rid of the error messages in this branch – they don't even make sense because they refer to bits of code that aren't there. E.g. it says 'body[0]: validations is not a permitted attribute', but there isn't a validations attribute within that first key.

Any thoughts? Could we try merging and see if that resolves it/changes the errors? 

<img width="1265" alt="CleanShot 2023-11-17 at 14 10 01@2x" src="https://github.com/storybookjs/storybook/assets/122215197/6b3afb4d-ba34-44ff-bd52-be6c8af2e2db">